### PR TITLE
change variable labeling for default grassmann pluecker ideal (Related to issue #4018)

### PR DESCRIPTION
**Pull request by danteluber**
Changing default variable labeling for Grassmann Pluecker ideal. Variables are now indexed by sets. Related to issue #4018.

### CODE REVIEW

**Review by lgoettgens — APPROVED** [review]
This enhances readability of the output by a lot!

**Comment by danteluber** [comment]
It appears I branched off of an old master creating merge conflicts. Will address this later

**Comment by lgoettgens** [comment]
I did my best to try to resolve the conflict of this with https://github.com/oscar-system/Oscar.jl/pull/3765. @danteluber please check if this is what you intended

File: src/Rings/mpoly-ideals.jl [hunk 1]
```diff
@@ -2063,23 +2063,25 @@ small_generating_set(I::MPolyIdeal{<:MPolyDecRingElem}; algorithm::Symbol=:simpl
...
 end
 
 @doc raw"""
     grassmann_pluecker_ideal([ring::MPolyRing,] subspace_dimension::Int, ambient_dimension::Int)
 
-Given a (possibly graded) ring, an ambient dimension, and a subspace dimension, return the ideal in the ring
-generated by the Plücker relations. If the input ring is not graded, return the ideal in the ring with the standard grading.
-If the ring is not specified return the ideal in a multivariate polynomial ring over the rationals with the standard grading.
+Given a (possibly graded) ring, an ambient dimension $n$ and a subspace dimension $d$, return the ideal in the ring
+generated by the Plücker relations.  If the input ring is not graded, return the ideal in the ring with the standard grading.
```
Comment (lgoettgens) [inline src/Rings/mpoly-ideals.jl]:
```suggestion
generated by the Plücker relations. If the input ring is not graded, return the ideal in the ring with the standard grading.
```

**Review by antonydellavecchia — APPROVED** [review]

**Pull request merged**